CONTRIBUTING.md

```diff
@@ -3,6 +3,17 @@
 Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. If you decide to contribute anything, please do the following:
 
 1. Fork this repository
+1. (Optional) Clone the fork
+   ```bash
```

Comment:
Just one minor suggestion more. Seems that if you add a newline here the render looks better, more separated from code block:

```suggestion

   ```bash
```



### Before this

![image](https://user-images.githubusercontent.com/23049315/102922142-d33d0080-448d-11eb-93db-a3bb3d1e8d39.png)


### After this

![image](https://user-images.githubusercontent.com/23049315/102922030-a852ac80-448d-11eb-9b92-ad3e644022c3.png)

Reply:
Hmm, not sure if I agree as the large spacing between list elements looks worse to me. But if other @simple-icons/maintainers or contributors agree with you I'm happy to change it :slightly_smiling_face:

Reply:
I personally prefer the larger spacing, but not a deal breaker either way for me.

Reply:
Alright, I updated it in 74408f27d74b61fed82e64f5ee90dfb3b31eed21 :+1: